.gitignore

```diff
@@ -2,6 +2,8 @@
 # SPDX-License-Identifier: Apache-2.0
 
 /build*/
+CMakeFiles
```

Comment:
We are ignoring those already by the previous line.

Builds should never be done directly in the source directory.